### ISSUE TITLE
Terraform-prometheus-ecs

### ISSUE THREAD

**Issue by guillermocol** [3h4x/terraform-prometheus-ecs#1]
Hello, 

Thank you for the creation of this code. I have a really quick one, what is the execution process of these terraform scripts?

Thank you for your help!
g

**Comment by 3h4x** [comment]
Hey @guillermocol !
Did you read related blog post https://3h4x.github.io/tech/2020/02/07/prometheus-on-ecs-poc.html ?

**Comment by vimox-shah-genea** [comment]
@3h4x yes I have read it but there is no instruction on how to run this PoC. It would be great If you can create a readme file and add details.
